templates/serviceAttributesContainer.hbs

```diff
@@ -0,0 +1,15 @@
+<section class='attributes-wrapper'>
+  <div class='container'>
+
+    <div class='container__attribute-section'>
+      {{#each attributes}}
+        {{{module this}}}
+      {{/each}}
+    </div>
+
+    <div class='container__line-divider'></div>
+
+  </div>
+
+  <a class='link-date'>{{{module bookNowButton}}}</a>

```

Comment:
If you put the `module bookNowButton` inside an `<a>` tag, you'll have two anchor links. Remove the `<a>` tag around and just leave the `{{{module bookNowButton}}}` code, and use the `.link` class that the `bookNowButton` has to specify your button styles.